lib/si-data-pg/src/lib.rs

```diff
@@ -2888,3 +2915,40 @@ impl PgOwnedTransaction {
 async fn test_connection_infallible_and_warm_up_pool_task(check_pool: PgPool) {
     let _result = check_pool.test_connection().await;
 }
+
+fn evict_old_and_idle_pgpool_connections(
+    pg_pool: Pool,
+    check_interval: Duration,
+    max_age: Duration,
+    max_idle_time: Duration,
+) -> impl Future<Output = ()> {
+    let min_check_interval = Duration::from_secs(30);
```

Comment:
I'm a little curious for the minimum value--is that for a little safety?

Reply:
Yeah, I wanted to make sure we definitely wouldn't configure the system to run this too much, since it blocks the pool. Just a bit of safety. I'm adding a warn! to make this clearer

Reply:
also added a comment to explain